Makefile

```
@@ -0,0 +1,63 @@
+# Makefile for agent-zero-docker-minimal
+# Google-style makefile for building and deploying Agent Zero
+
+.PHONY: build run stop clean help deploy
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)